build_tools/github_actions/build_dist.py

```diff
@@ -179,17 +179,48 @@ def build_py_runtime_pkg():
       f"-DIREE_BUILD_PYTHON_BINDINGS=ON",
       f"-DIREE_BUILD_SAMPLES=OFF",
       f"-DIREE_BUILD_TESTS=OFF",
-  ])
+  ],
+                 check=True)
 
   print("*** Building ***")
-  subprocess.check_call([
+  subprocess.run([
       sys.executable,
       CMAKE_CI_SCRIPT,
       "--build",
       BUILD_DIR,
       "--target",
       "install-IreePythonPackage-rt-stripped",
-  ])
+  ],
+                 check=True)
+
+
+def bazel_build_tf_binary(target):
+  """Builds a binary in the IREE-TF Workspace and returns the filepath."""
+
+  # Builds a runnable target and returns the path to the executable. Yes this is
+  # really the best Bazel gives us.
+  # See https://github.com/bazelbuild/bazel/issues/8739
+  cmd = [
+      "bazel",
+      "run",
+      "--run_under=echo",
```

Comment:
WTF. I swear, the people who wrote these things never tried to use them in a real project.

Reply:
Left a comment on the issue.